### PR TITLE
Duplicates Africa's Talking Balance to the top of the screen 

### LINKED ISSUE #306: Move/Duplicate Africa's Talking Balance to the top of the screen
This way users are more likely to actually check it routinely.

### PR DESCRIPTION
**Pull request by DanielMwendwa**
Closes #306

### CODE REVIEW

File: public/js/controllers/ui_controller.js [hunk 1]
```diff
@@ -189,28 +189,47 @@ export class UIController {
         UIController.resetUI();
         document.head.appendChild(UIController.getScrollJsScript());
         let html = `<div class="container"> 
-            <div class="d-md-flex justify-content-between p-1">
+            <div class="d-md-flex justify-content-between">
+                <div class="d-flex">
+                    <span class="font-weight-bold txt-brown mr-1">Africa's Talking Units Balance:</span>
+                    <div class="font-weight-bold mb-0" id="AT-units-bal"></div>
```
Comment (IsaackMwenda) [inline public/js/controllers/ui_controller.js:197]: Moving this to the top is a good idea, however, the UI now looks a bit untidy and congested. I would create a new card above with three columns and populate them with;
- Project Name 
- Africas Talking Bal
- Last Updated time
The balance the spacing between the time filters and the above card.

**Review by IsaackMwenda — CHANGES_REQUESTED** [review]
Looks good - requesting changes due to a couple UI suggestions.

File: public/js/controllers/ui_controller.js (hunk 1 above)
Comment (DanielMwendwa) [reply public/js/controllers/ui_controller.js:197]: I've done some changes, suggestions/feedback are welcome.

**Review by IsaackMwenda — APPROVED** [review]
Looks better now - please remove the duplicate project details tab at the bottom.

**Comment by DanielMwendwa** [comment]
> Looks better now - please remove the duplicate project details tab at the bottom.

Updated, thanks.

**Pull request merged**